diff --git a/h264_encoder_core/CMakeLists.txt b/h264_encoder_core/CMakeLists.txt
index ce8edec..b28ffdf 100644
--- a/h264_encoder_core/CMakeLists.txt
+++ b/h264_encoder_core/CMakeLists.txt
@@ -67,20 +67,20 @@ INSTALL(EXPORT ${PROJECT_NAME}-targets DESTINATION share/${PROJECT_NAME}/cmake)
 ## Tests ##
 #############
 
-enable_testing()
-find_common_test_packages()
-add_common_gtest(test_h264_encoder_core
-  test/h264_encoder_test.cpp
-)
-if(TARGET test_h264_encoder_core)
-  target_include_directories(test_h264_encoder_core PRIVATE
-    include
-    ${aws_common_INCLUDE_DIRS}
-  )
-  target_link_libraries(test_h264_encoder_core
-    ${PROJECT_NAME}
-    ${aws_common_LIBRARIES}
-    ${GMOCK_LIBRARY}
-    pthread
-  )
-endif()
+#enable_testing()
+#find_common_test_packages()
+#add_common_gtest(test_h264_encoder_core
+#  test/h264_encoder_test.cpp
+#)
+#if(TARGET test_h264_encoder_core)
+#  target_include_directories(test_h264_encoder_core PRIVATE
+#    include
+#    ${aws_common_INCLUDE_DIRS}
+#  )
+#  target_link_libraries(test_h264_encoder_core
+#    ${PROJECT_NAME}
+#    ${aws_common_LIBRARIES}
+#    ${GMOCK_LIBRARY}
+#    pthread
+#  )
+#endif()
